src/commonMain/kotlin/com/algolia/search/model/recommendation/FacetScoring.kt

```diff
@@ -0,0 +1,22 @@
+package com.algolia.search.model.recommendation
+
+import com.algolia.search.serialize.KeyFacetName
+import com.algolia.search.serialize.KeyScore
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/**
+ * Configure the importance of facets
+ */
+@Serializable
+public data class FacetScoring(
+
+    /**
+     * Attribute name
+     */
+    @SerialName(KeyFacetName) val facetName: String,
+    /**
+     * Score for the facet
+     */
+    @SerialName(KeyScore) val score: Int
+)
```

Comment:
nit: the Kotlin's formater doesn't add new line at EOF? Generally speaking it's a [good practice](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file) to have them.

Reply:
@Ant-hem good point, @Aallam will work on adding a linter soon to the client and we could fix that 😄